csharp/extractor/Semmle.Extraction.CIL.Driver/ExtractorOptions.cs

```diff
@@ -237,43 +228,25 @@ private void AddFileOrDirectory(string path)
         /// </summary>
         public IEnumerable<AssemblyName> MissingReferences => assemblyList.MissingReferences;
 
-        private void ParseArgs(string[] args)
+        public override bool HandleFlag(string flag, bool value)
         {
-            foreach (var arg in args)
+            switch (flag)
             {
-                if (arg == "--verbose")
```

Comment:
The semantics of the --verbose flag has now been changed by using the base.HandleFlag method. Is this intentional?

Reply:
Good spot; I think it makes sense that `--verbose` means the same thing for all `.exe`s.